### ISSUE TITLE
Promote the aux?

### ISSUE THREAD

**Issue by nschneid** [UniversalDependencies/UD_English-EWT#301]
In [http://match.grew.fr/?corpus=UD_English-EWT@dev&custom=61f5ade50e501](these examples), the nominal has an `aux` dependent but no `cop`. The `aux` should probably be promoted to head of the clause per the guidelines for ellipsis.

**Comment by nschneid** [comment]
(this is similar in spirit but not totally redundant with the queries in #302)

**Comment by nschneid** [comment]
- [ ] It may also be **privileged**/ADJ or otherwise protected/VERB by work product immunity or other legal rules. 

The initial words in the sentence are attached to "privileged" as if it were a verb; it may have been changed to ADJ. ADJ seems a better fit semantically (if something has an legal status of "privileged" that doesn't necessarily mean there was an act of privileging it), but that places an ADJ in coordination with a VERB, creating a tension between `nsubj`/`nsubj:pass` for "it" and `aux`/`aux:pass` for "be". Thoughts, @amir-zeldes?

**Comment by nschneid** [comment]
(results not yet addressed: tricky relative constructions, and accidentally omitted words)

**Comment by amir-zeldes** [comment]
This is mainly trick to me due to the by phrase, which, if it scopes over "privileged" too, suggests that it's a passive participle. But taken as a whole, I don't think it's a participle. I think PTB test like negation (non-privileged) and intensification (very privileged) suggest it's an adjective, in which case it's an unlike-coordination, and the rule of "first dibs" applies. If this is accepted (privileged=ADJ) then the guidelines are clear: in basic dependencies it would have to be nsubj+cop (not passive or aux), and in enhanced propagation we could still decide that "It" is also a passive subject of "protected". That only leaves the question of how to attach the by phrase: if it attaches to privileged, then that is just a "means" by-PP (and if we propagate it to "protected" as obl:agent, then we are saying it's zeugmatic, which is at least structurally a valid UD analysis)

**Comment by nschneid** [comment]
I agree the attachment of the by-PP isn't completely clear but I think attaching it just to "protected" is a reasonable analysis. (One could insert a comma before "or" without doing violence to the meaning, I think).
